Automattic/wp-calypso
# [Domains] Consolidate featured suggestion styling

Extracted refactoring changes from #24515.

This change consolidates all featured suggestion styles into `/components/domains/featured-domain-suggestions/`. Unnecessary or redundant styles were removed (i.e. specifying `flex` along with `flex-grow`). The interface remains visually identical.

<img width="980" alt="wordpress_com" src="https://user-images.githubusercontent.com/4044428/39387429-c72b4300-4a36-11e8-96a0-bc6df0890f1b.png">

# Testing instructions
1. Spin up this branch locally.
2. Navigate to `/start/domains`
3. Ensure that the interface renders as expected.
